run_tests.sh

```diff
@@ -143,8 +143,6 @@ else
                          "tests/defences/test_rounded.py" \
                          "tests/defences/test_thermometer_encoding.py" \
                          "tests/defences/test_variance_minimization.py" \
-                         "tests/defences/detector/evasion/subsetscanning/test_detector.py" \
```

Comment:
It looks like the new tests are not yet being run. Please change lines 22 and 23 of this file to:
```python
pytest --cov-report=xml --cov=art --cov-append  -q -vv tests/defences/detector/evasion --framework=$framework --durations=0
    if [[ $? -ne 0 ]]; then exit_code=1; echo "Failed defences/detector/evasion tests"; fi
```